File: Atcoder/Ruby/180.ABC/B.rb
kes  = gets.to_i
mori = gets.split(" ").map &:to_i

yama = 0
mori.each do |i|
  yama += i.abs
end
puts yama

evis = 0
mori.each do |i|
  evis += (i*i).abs
end
yama = Math.sqrt(evis)
p yama

puts mori.map(&:abs).max

File: Atcoder/PHP/180.ABC/B.php
<?php
fscanf(STDIN, "%d", $kes);
$map = explode(" ", trim(fgets(STDIN)));

$moriyama = 0;
for ($i = 0; $i <= count($map); $i++) {
    $moriyama += abs($map[$i]);
}
echo $moriyama . PHP_EOL;

$fuji = 0;
for ($i = 0; $i <= count($map); $i++) {
    $fuji += abs($map[$i] ** 2);
}
echo sqrt($fuji) . PHP_EOL;

$ryo = [];
for ($i = 0; $i <= count($map); $i++) {
    $ryo[] = abs($map[$i]);
}
echo max($ryo);